src/fastmean.c

```diff
@@ -48,21 +48,24 @@ SEXP fastmean(SEXP args)
   if (narm) {
     switch(TYPEOF(x)) {
     case LGLSXP:
-    case INTSXP:
+    case INTSXP: {
+      int *pi = INTEGER(x);
       for (i = 0; i<l; i++) {
-        if(INTEGER(x)[i] == NA_INTEGER) continue;
-        s += INTEGER(x)[i];   // no under/overflow here, s is long double not integer
+        if(pi[i] == NA_INTEGER) continue;
```

Comment:
- minor thing but you could also adjust spacing here `if (...`
- `continue` could go into new line so code coverage can test that